Modules/Utility/Table.lua

```diff
@@ -36,23 +36,11 @@ end
 function lib.ToList(tab)
 	local list = {}
 	for _, item in pairs(tab) do
-		table.insert(list, item)
+		list[#list+1] = item
 	end
 	return list
 end
 
---- Counts the number of items in `tab`.
--- Useful since `__len` on table in Lua 5.2 returns just the array length.
--- @tparam table tab Table to count
--- @treturn number count
-function lib.Count(tab)
```

Comment:
`lib.Count` works on  non-array components as the document specifies. We can't remove this method.